README.md

```diff
@@ -125,6 +125,53 @@ The aim of this lammbda is to have up to date metrics of yld's open source contr
 
 Main [`@yld.io/oss-stats`](https://www.npmjs.com/package/@yldio/oss-stats) to aggregate all open source contributions for `yldio` organization members.
 
+## Styling
+
+### theme
+
+The whole application is wrapped in a theme file `src/utils/theme.js` that contains the official breakpoints, spacing and colors that can be used within the app. It's always available within a styled component as `theme` prop.
+
+### Grid
+
+We often use a system of Grid, Row and Column; all defined within `src/components/grid.js` to style sections. They can be confusing at first. The Grid component is a div that automatically defines how a section behaves accoding to the official breakpoints of the app.
```

Comment:
This is a bit misleading. The site is designed using a grid system

Reply:
Updated